R/aaa.R

```diff
@@ -9,7 +9,7 @@ format_root_section <- function() {
     "starting with the current working directory during package load time,",
     "the directory hierarchy is walked upwards ",
     "until a directory with at least one of the following conditions is found:",
-    gsub('"([^"]+)"', "`\\1`", format_root_criteria_items()),
+    format_root_criteria_items(),
```

Comment:
I just found this easier to reason about by relocating the `gsub()` below. Now we do that while each criterion's description is still its own element. This also limits the damage an undesirable regex substitution can do, i.e. limits it to a single item versus communicating across the whole string (that was actually what caused the `.svn` weirdness before).